File: archive/classic/FizzBuzz/Lambda.rb
def fizzBuzz(n)
  word = ""

  word += "Fizz" if (n % 3).zero?
  word += "Buzz" if (n % 5).zero?

  word = n.to_s if word.empty?

  word
end

if __FILE__ == $PROGRAM_NAME
  1.upto(100) do |i|
    puts(fizzBuzz(i))
  end
end

File: archive/classic/FizzBuzz/Lambda.js
const fizzBuzz = (n) => {
  let word = '';

  if (n % 3 === 0) {
    word += 'Fizz';
  }
  if (n % 5 === 0) {
    word += 'Buzz';
  }

  if (word.length === 0) {
    word = String(n);
  }

  return word;
};

const main = () => {
  for (let i = 1; i <= 100; i++) {
    console.log(fizzBuzz(i));
  }
};

main();
